x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -63,300 +129,493 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
-                  ),
-                  'data-test-subj': 'mlModelsStartDeploymentModalNormalPriority',
-                },
-              ]}
-              data-test-subj={'mlModelsStartDeploymentModalPriority'}
-            />
-          </EuiFormRow>
-        </EuiDescribedFormGroup>
-      ) : null}
+      }
+    } else if (
+      (cloudInfo.isCloud && !cloudInfo.isMlAutoscalingEnabled) ||
+      (!cloudInfo.isCloud && showNodeInfo)
+    ) {
+      // Running in cloud with autoscaling disabled or on-prem
+      if (config.adaptiveResources) {
+        // With adaptive resources
+        switch (config.vCPUUsage) {
+          case 'low':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.lowCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to {maxVCPUs, plural, one {vCPU} other {vCPUs}}, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+                values: { maxVCPUs: vcpuRange.max },
+              }
+            );
+          case 'medium':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.mediumCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to {maxVCPUs, plural, one {vCPU} other {vCPUs}}, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+                values: { maxVCPUs: vcpuRange.max },
+              }
+            );
+          case 'high':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.highCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level may use {maxVCPUs, plural, one {vCPU} other {vCPUs}} - the maximum number of vCPUs available for this deployment. If the maximum is 2 vCPUs or fewer, this level is equivalent to the medium or low level.',
+                values: { maxVCPUs: vcpuRange.max },
+              }
+            );
+        }
+      } else {
+        // Without adaptive resources
+        switch (config.vCPUUsage) {
+          case 'low':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.lowCpuStaticHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to the minimum required for ELSER to run. It may not be sufficient for a production application.',
+              }
+            );
+          case 'medium':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.mediumCpuStaticHelp',
+              {
+                defaultMessage: 'Your model will consume {staticVCPUs} vCPUs consistently.',
```

Comment:
```suggestion
                defaultMessage: 'Your model will consume {staticVCPUs} vCPUs, even when not in use.',
```

Reply:
Updated in c3077e11b02d01f3be0acf409a86f4fc0cbb4a43